.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
Looks nice, I would suggest taking a look at this as well. https://github.com/dotnet/runtime/blob/main/.github/ISSUE_TEMPLATE/01_bug_report.yml

Reply:
Ah yes this is good!

Reply:
@srcnalt should I change to something more like dotnet now?

Reply:
Not exactly, its more of a Unity thing tbh things we wanna know if error message if any, versions of unity and SDK modules, rendering pipeline, other external non unity plugins (photon etc), maybe operating system, and which platform they were building to

Reply:
I think I'll just leave for now. We can do a 2nd pass on these later (maybe our docs guy can take a look over it)